client/app/hostBill/hostBill.controller.js

```diff
@@ -80,12 +82,30 @@
       }
     };
 
+    self.getFeePercent = function () {
```

Comment:
hey @Faline10 I noticed there was a lot of repetition in terms of `getTip()` and `getTipPercent()`, could you refactor so that there's only 2 functions instead of 4? getDiscountPercent and getFeePercent could also be incorporated as well